### ISSUE TITLE
[Bug]: local API-ERROR: SyntaxError: Missing catch or finally after try

### ISSUE THREAD

**Issue by Cyborgscode** [FreeTubeApp/FreeTube#3402]
### Guidelines

- [X] I have encountered this bug in the [latest release of FreeTube](https://github.com/FreeTubeApp/FreeTube/releases).
- [X] I have searched the issue tracker for [open](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aopen+is%3Aissue) and [closed](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aissue+is%3Aclosed) issues that are similar to the bug report I want to file, without success.
- [X] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the bug I want to file, without success.
- [X] This issue contains only one bug.

### Describe the bug

1.  Abo's
2. click on any presented video
![Bildschirmfoto vom 2023-04-05 22-08-36](https://user-images.githubusercontent.com/8317797/230199567-354da7ba-b4ea-4d60-a890-35eb467d55a6.png)

### Expected Behavior

I believe, a new version is needed.

### Issue Labels

API issue, content not loading

### FreeTube Version

0.18.0 Beta

### Operating System Version

Fedora Linux 36

### Installation Method

.rpm

### Primary API used

Local API

### Last Known Working FreeTube Version (If Any)

0.18.0 beta

### Additional Information

Bug started 2 days ago. Before this date, everything worked as expected.

### Nightly Build

- [ ] I have encountered this bug in the latest [nightly build](https://docs.freetubeapp.io/development/nightly-builds).

**Comment by pm4rcin** [comment]
It started today for me and now I can't play any video even with Invidious proxy. I have freetube flatpak.
![s](https://user-images.githubusercontent.com/37148802/230213833-cba5c1c4-6c9f-4726-a6b5-0d42e90ef654.png)

**Comment by FondsofFall** [comment]
Same...same... except I use AppImage. Thought the solution would be: by deleting the AppImage then "reinstall" it. 

The closed caption is no longer working after seeing " Missing catch or finally after try"  It did not give me a chance to click a copy of error but thank you for report a bug on Github. :)

**Comment by skaendo** [comment]
Same just started happening to me today as well, now nothing is playing at all. Debian package.

**Comment by benw01** [comment]
plus one, happening for me too. Version 0.18.0 (0.18.0), MacOS 13.3.

**Comment by johnno66** [comment]
Same here, started yesterday after some kind of YT outage.   Fallback to Invidious works, but with limited functionality.
Fedora37,  Freetube  0.18.0

**Comment by trenton4k** [comment]
work on freetube-0.18.0-nightly-2844 on windows

**Comment by Cyborgscode** [comment]
Confirmed: Build 2844 fixes the issue.

**Comment by johnno66** [comment]
I can also confirm freetube-0.18.0-nightly-2844.x86_64.rpm fixes the issue.
Thanks for the quick solution!

**Comment by benw01** [comment]
Confirmed that build 2844 fixes this issue on MacOS (freetube-0.18.0-nightly-2844.dmg)

**Comment by FondsofFall** [comment]
Update:  The screen is still appears "Local API Error (Click to copy): SyntaxError: Missing catch or finally after try" and "Falling back to Invidious API" and sometimes " Invidious API Error(Click to copy): undefined" The video is loaded but closed caption is not working properly.  The good news is that I figured it out by changing current Invidious Instance then load the video, and press "C" on keyboard for turning caption on. For some reason, it does not work when you click on icon CC. Oh, well.  Oh! Important to press ONCE because if you press "C" more than once then the caption repeated of same sentence.

**Comment by fuengfah** [comment]
> work on freetube-0.18.0-nightly-2844 on windows

where I can get Nightly buid?

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
@fuengfah see https://docs.freetubeapp.io/development/nightly-builds for more info on the nightlys

**Comment by pntc** [comment]
@trenton4k thanks! Confirmed, working on Linux as well.

**Comment by logos88** [comment]
I have the same issue in the freetube-bin from AUR

**Comment by Growebis** [comment]
I also have the same issue with Linux Mint, Freetube 0.18.0 beta

**Comment by RevAngel7** [comment]
Same here for the .deb version on ubuntu 22.04 (latest mainline). My invidious session had to be changed to a healthy one to at least work and play the video. But the local API seems to be dead for the v0.18.0 Beta version.

**Comment by Cyborgscode** [comment]
For a quick solution: **Install the nightly version as explained above!**

**Comment by RevAngel7** [comment]
> For a quick solution: **Install the nightly version as explained above!**

Done and working, thank you!

**Comment by pm4rcin** [comment]
Changing instance is also a viable option for the time being. It worked for me without needing to download nightly.

**Comment by zestygrass** [comment]
Same exact issue, Invidious fallback was making it work for last few days.
Just went ahead and installed #2846 nightly build, local API is now functioning just fine.

I was using freetube-bin from AUR, so modified the PKGBUILD a bit to install the nightly version, didn't notice any issues yet.

**Comment by SJRouge91** [comment]
Was having the same error as well last 3-4 days, and after installing nightly build the descriptions in the videos are showing up again, and everything is fine. Using the portable 64-bit version on Windows. Thanks all :)

**Comment by RevAngel7** [comment]
Yes, a big "thank you" to all maintainers of this project! If you are a regular user of FreeTube, please remember these projects also keep alive because of your financial donations.
https://github.com/FreeTubeApp/FreeTube#donate

**Comment by RevAngel7** [comment]
Well, looks like youtube does not like FreeTube any more. Error 400 on the API key.
![FreeTube_API_error_400](https://user-images.githubusercontent.com/38556983/230887308-175673b4-9fc5-4d38-9323-247c7cc1d772.jpg)

**Comment by Cyborgscode** [comment]
> Well, looks like youtube does not like FreeTube any more. Error 400 on the API key. ![FreeTube_API_error_400](https://user-images.githubusercontent.com/38556983/230887308-175673b4-9fc5-4d38-9323-247c7cc1d772.jpg)

Pls open a new Issue for that error.

Local API works fine here with 2844

**Comment by RevAngel7** [comment]
The error is from v0.18.0-nightly-2844 Beta. I am opening a new issue with it.

**Comment by RevAngel7** [comment]
https://github.com/FreeTubeApp/FreeTube/issues/3418
